lib/overcommit/hook_runner.rb

```diff
@@ -74,7 +74,13 @@ def run_hooks
 
         print_results
 
-        !(@failed || @interrupted)
+        hook_failed = @failed || @interrupted
+
+        if hook_failed && @context.respond_to?(:post_fail_message)
+          puts @context.post_fail_message
```

Comment:
The `printer` should probably be used here, right? I wasn't sure yet how to use the `printer` for this, so I wanted to commit and get a discussion going first, since this is my first ruby PR.

Reply:
I couldn't find a test for `run_hooks` so I wasn't sure how to test this. Happy to add a test if I could be pointed to where?

Reply:
Yes, let's use the printer class (and give it a specific method for this case). Thanks!